### PR TITLE
Rework configuration tutorial with docker [ECR-697]

### PR DESCRIPTION
**Pull request by vitvakatu**

### CODE REVIEW

File: services/configuration/docker/example-start.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Usage:
+# example-start.sh <number of nodes>
+
+node_count=$1
+port_start=8000
+port_end=$((port_start + (2 * node_count) - 1))
+
+docker run -p ${port_start}-${port_end}:${port_start}-${port_end} vitvakatu/exonum-configuration-example:latest $node_count
```
Comment (aleksuss) [inline services/configuration/docker/example-start.sh]: Is this a right path `vitvakatu/` ?

File: services/configuration/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM rust:1.23.0
+
+WORKDIR /usr/src
```
Comment (aleksuss) [inline services/configuration/docker/Dockerfile]: Why would you prefer to use a path `/usr/src` ?

File: services/configuration/doc/testnet-api-tutorial.md [hunk 3]
```diff
@@ -12,106 +38,48 @@ cargo install --example configuration
 `exonum` crate system dependencies and rust toolchain configuration -
 [exonum install instructions](https://github.com/exonum/exonum/blob/master/INSTALL.md).
 
-## Running testnet
+#### Generate testnet directory and config
+
+`4` is a required indexed parameter and stands for the number of nodes in
```
[the remark this answers is not in the document]
Comment (vitvakatu) [reply services/configuration/doc/testnet-api-tutorial.md]: Seems reasonable, but I can't imagine how to explain the distribution of ports for every node. Will try though

File: services/configuration/docker/Dockerfile (hunk 2 above)
Comment (vitvakatu) [reply services/configuration/docker/Dockerfile]: Well, actually, I don't know. I've used the example from official rust docker image. Is there any drawbacks?

Comment (aleksuss) [reply services/configuration/docker/Dockerfile]: Actually this path is used for storing some source files. Linux kernel sources e.g. I'd prefer some thing in `home` directory. E.g. `/home/exonum` Basically it is not important. Just an advice.

**Pull request merged**